scripts/configure_dev.sh

```diff
@@ -32,6 +58,11 @@ elif [ "${OS}" = "darwin" ]; then
     install_or_upgrade autoconf
     install_or_upgrade automake
     install_or_upgrade shellcheck
+    install_or_upgrade python3
+fi
+
+if [[ ${SKIP_GO_DEPS} == "True" ]]; then
```

Comment:
Bash supports booleans, you can just do `if $SKIP_GO_DEPS`

Reply:
OK, will convert to booleans.